### ISSUE TITLE
Please provide binaries for download on GitHub Releases

### ISSUE THREAD

**Issue by probonopd** [satoshinm/pill_serial#6]
Please provide binaries for download on GitHub Releases (or another download URL), thanks.

**Comment by zoobab** [comment]
+1

**Comment by caniggia1** [comment]
+1

**Comment by trysomethingelse** [comment]
+1

**Comment by dalbert2** [comment]
I'm not sure how to add it to releases, but here's the binary built using cygwin.  It seems to work as advertised (enumerates as a Black Magic Probe and creates 3 sequential virtual serial ports):

[pill_serial.zip](https://github.com/satoshinm/pill_serial/files/2719841/pill_serial.zip)

**Comment by satoshinm** [comment]
Agreed publishing binaries would be a good idea. @dalbert2 I've invited you as a collaborator if you want to publish yours in this repo under GitHub releases. To do so create a tag in git then draft a new release on GitHub: https://help.github.com/articles/creating-releases/ and it'll let you attach files (although attaching the .zip to an issue comment works too).

**Comment by piyushverma** [comment]
Very useful, I just flash and test it works, before I build and flash but it did not work.
Any idea what could be wrong ? I am using ubuntu 16.04, just clone git, and install required gcc using apt-get install and it build successful. but unfortunately it did not run, usb not detected
